src/components/NewTable/components/TablePagination/TablePagination.tsx

```diff
@@ -73,16 +75,18 @@ export const TablePagination = ({ currentPage = 1, totalPages, onChange, visible
         </TablePaginationButton>
       ))}
       <TablePaginationButton
+        background
         disabled={currentPage === totalPages}
         onClick={onChange ? () => onChange(currentPage) : undefined}
       >
-        <KeyboardArrowRight className={styles.icon} />
+        <KeyboardArrowRight fontSize={'small'} className={styles.icon} />
       </TablePaginationButton>
       <TablePaginationButton
+        background
         disabled={currentPage === totalPages}
         onClick={onChange ? () => onChange(totalPages - 1) : undefined}
       >
-        <KeyboardDoubleArrowRight className={styles.icon} />
+        <KeyboardDoubleArrowRight fontSize={'small'} className={styles.icon} />
```

Comment:
here and above (lines 58, 65, 82, 89) I think can be just `fontSize="small"`

Reply:
I actually prefer using the brackets as this is jsx. It's just personal preference though, we can align on that :)